### ISSUE TITLE
Who has got network / Distriubuted features right?

### ISSUE THREAD

**Issue by apurbas** [hyperion-project/hyperion#606]
I have been experimenting with vl42 Hyperion forward to Philips Hue on various OS (Openelec, OSMC, jessy) with limited/ no success.Has anyone one of you got  it absolutely right on Raspberry Pi A?  

I'm sure some of us has got it right. If you are one among them, I need to know -

1. Which Raspberry model  are you using?
2.Which OS & version (Openelec, OSMC or Raspbian)?
3.Which Philips Hue bridge you are on ( Old/round or New/Square)?

Looking forward.

**Comment by tpmodding** [comment]
https://github.com/hyperion-project/hyperion/issues/594
https://github.com/hyperion-project/hyperion/issues/578
https://github.com/hyperion-project/hyperion/issues/556

Don't you wont to close the other issues?

**Comment by apurbas** [comment]
@tpmodding - good question. Would love to close that but not in the bug finding / recompiling hyperion way. That's tiresome considering that i'm not command prompt savvy. So if these features have worked for someone here - I would love to test that on my setup & contribute.

**Comment by tpmodding** [comment]
i have no hue sorry, but in the german rpi forum some guys use this option and it works flawless for them

http://www.forum-raspberrypi.de/Forum-ambilight, think it is only a config thing...

**Comment by apurbas** [comment]
Hey @tpmodding - thanks buddy for the reference.

 I have gone through Doc.ex conversation here http://www.forum-raspberrypi.de/Thread-hyperion-hyperion-mit-forwarder-funktion-update-27-02-2016?page=4

And created configs with vl42 grabber frame decimation (2)/size decimation (1) changes at the master side & device transition time at the client/hue side (10). Works with a lag for sometime then throws segmentation fault or array error etc. Its not reliable for sure.

**Comment by tpmodding** [comment]
" Its not reliable for sure." yes, i heard that the hue cannot utilize to many frames... and we can not change the frames for the forwarder (?) so it uses the frames form "master"

but it works for now with this little problem right?

**Comment by apurbas** [comment]
Works for a "little" with this little problem :-) ..as I said I'm getting segmentation fault/ array error.

**Comment by ntim** [comment]
As I pointed out in #578, could you please produce a stack trace to see where the error is coming from exactly?

**Comment by apurbas** [comment]
Is there any similar guide for Openelec? 
https://github.com/hyperion-project/hyperion/blob/master/CompileHowto.txt

**Comment by tpmodding** [comment]
same binaries...? dont need another guide..

**Comment by brindosch** [comment]
You cant compile on oe
Am 22.04.2016 4:51 nachm. schrieb "tpmodding" <notifications@github.com>:

**Comment by apurbas** [comment]
Thank you @brindosch. If among 3 we have so much confusion, imagine others outside this conversation ;-) Jokes apart - As we extend more support to Hue, we seriously need to fix this across all OS. As I said - I'm not very pro with command prompts & recompiling hyperion etc , its tough for me actually.

**Comment by brindosch** [comment]
@apurbas 
Could you install the latest beta and you might catch up the information for your fault

**Comment by brindosch** [comment]
@ntim 
your advice and hue wisdom is appreciated :)
https://hyperion-project.org/threads/hue-switch-blue-light.129/

**Comment by ntim** [comment]
@brindosch 
On it. Also had something similar but only once every 1-2 minutes which was ultimately everytime resolved by a restart but maybe this develops into a separate issue. Maybe it occurs with other drivers too but since the hue is so slow it becomes apparent.

**Comment by brindosch** [comment]
So we have some hue users and not sure why, you are currently one of few people that have problems?
proceed with
https://github.com/hyperion-project/hyperion/issues/578

This information would really help here
/closed
